content/en/docs/armory-admin/armory-agent/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Armory Agent for Kubernetes"
+linkTitle: "Armory Agent"
+weight: 4
+description: >
+  How the Armory Agent works
```

Comment:
Since this is the top lvl page for Agent, I would expect it to explain what Agent is and why I would want to use it in my environment, before the actual inner workings.

> The Armory Agent is a lightweight service that connects back to Spinnaker’s Clouddriver service and streams changes from the Kubernetes clusters it is configured to watch.

That doesn't mean anything to a user because it doesn't answer what problem is this solving/why would someone want to use this.

Reply:
I expect users at that point to have read the intro and `Administration` to be for more advanced users who definitely would understand the sentence. Perhaps it'd make more sense if consolidated like you suggested further down.